base/nvti.c

```diff
@@ -1057,6 +1057,46 @@ nvti_set_solution_type (nvti_t *n, const gchar *solution_type)
   return (0);
 }
 
+/**
+ * @brief Add a tag to the NVT tags.
+ *
+ * @param n The NVT Info structure.
+ *
+ * @param name  The tag name. A copy will be created from this.
+ * @param value The tag value. A copy will be created from this.
+ *
+ * @return 0 for success. Anything else indicates an error.
+ */
+int
+nvti_add_tag (nvti_t *n, const gchar *name, const gchar *value)
+{
+  char *newtag;
```

Comment:
```suggestion
  gchar *newtag;
```

gchar because that's what g_strconcat returns.

Reply:
Done.